freqtrade/rpc/rpc.py

```diff
@@ -849,16 +849,19 @@ def _rpc_reload_config(self) -> dict[str, str]:
         self._freqtrade.state = State.RELOAD_CONFIG
         return {"status": "Reloading config ..."}
 
-    def _rpc_stopentry(self) -> dict[str, str]:
+    def _rpc_pause(self) -> dict[str, str]:
         """
-        Handler to stop buying, but handle open trades gracefully.
+        Handler to pause trading (stop entering new trades), but handle open trades gracefully.
         """
         if self._freqtrade.state == State.RUNNING:
-            # Set 'max_open_trades' to 0
-            self._freqtrade.config["max_open_trades"] = 0
-            self._freqtrade.strategy.max_open_trades = 0
+            self._freqtrade.state = State.PAUSED
+            return {"status": "pausing trader ..."}
```

Comment:
I'm not sure i like this "return" here.
i understand the intend - but it hides the essential info (how can you start it again?).
it's a direct message that's returned - but wording of the fallthrough (L864) would be clearer as to what happened - so maybe we should remove this line in favor of that. (L862 is fine i think - as that's a different state change).

![image](https://github.com/user-attachments/assets/ae5cc145-bb1d-49ca-8fca-4452cf32bdaf)

Reply:
I've added the **Run /start to enable entries**. I also created a small wording difference when the bot just change to paused state or if it is already paused and the user trigger again `/pause`